google-chart.html

```diff
@@ -1,7 +1,7 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-ajax/iron-request.html">
 <link rel="import" href="../promise-polyfill/promise-polyfill-lite.html">
-<script type="text/javascript" src="https://www.google.com/jsapi"></script>
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
this element should really be loading the lib as a dep using https://github.com/GoogleWebComponents/google-apis/blob/master/google-legacy-loader.html

Then use it in the element:
`<google-legacy-loader library-url="https://www.gstatic.com/charts/loader.js?callback=%%callback%%" on-api-load="loaded"></google-legacy-loader>`

Does the new endpoint support a callback?